composer.json

```diff
@@ -49,15 +49,16 @@
 	},
 	"require-dev": {
     "pressbooks/coding-standards": "^1.1",
-		"codeception/module-db": "^1.1",
-		"codeception/module-webdriver": "^1.2",
-		"codeception/module-phpbrowser": "^1.0",
-		"codeception/module-asserts": "^1.3",
-		"codeception/module-filesystem": "^1.0",
-		"codeception/module-cli": "^1.1",
-		"codeception/util-universalframework": "^1.0",
-		"lucatume/wp-browser": "^3.0",
-		"yoast/phpunit-polyfills": "^1.0"
+    "codeception/module-db": "^1.1",
+    "codeception/module-webdriver": "^1.2",
+    "codeception/module-phpbrowser": "^1.0",
+    "codeception/module-asserts": "^1.3",
+    "codeception/module-filesystem": "^1.0",
+    "codeception/module-cli": "^1.1",
+    "codeception/util-universalframework": "^1.0",
+		"phpunit/phpunit": "^7.5",
```

Comment:
restores PHPUnit 7.5 as dev dependency for use by local devs until WP 5.9 is released, when we can revise this approach